Source/WTF/wtf/text/StringImpl.h

```diff
@@ -305,6 +305,8 @@ class StringImpl : private StringImplShape {
     bool is8Bit() const { return m_hashAndFlags & s_hashFlag8BitBuffer; }
     ALWAYS_INLINE const LChar* characters8() const { ASSERT(is8Bit()); return m_data8; }
     ALWAYS_INLINE const UChar* characters16() const { ASSERT(!is8Bit() || isEmpty()); return m_data16; }
+    ALWAYS_INLINE std::span<const LChar> span8() const { ASSERT(is8Bit()); return { m_data8, m_length }; }
```

Comment:
Why not call `characters8` to avoid writing the assertion twice?

Reply:
Good idea, will fix.